src/plugins/liveobjects/livemap.ts

```
@@ -174,12 +185,63 @@ export class LiveMap extends LiveObject<LiveMapData> {
+      }
+      if (currentEntry.tombstone === false && newEntry.tombstone === true) {
+        // current prop is not tombstoned, but new is. it means prop was deleted
+        update.update[key] = 'deleted';
+        continue;
+      }
+
+      // if tombstones are matching, need to compare values with deep equals to see if it was changed
+      const valueChanged = !deepEqual(currentEntry.data, newEntry.data, { strict: true });
+      if (valueChanged) {
+        update.update[key] = 'updated';
+        continue;
```

Biome

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)